testtools/tests/matchers/test_const.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2016 testtools developers. See LICENSE for details.
+
+from testtools import TestCase
+from testtools.matchers import Not, always, never
+
+
+class Test_ConstantMatches(TestCase):
+    def test_always(self):
+        self.assertThat(object(), always())
+
+    def test_never(self):
+        self.assertThat(object(), Not(never()))
```

Comment:
The more normal way to write tests for matchers is to use the `TestMatchersInterface` mixin. I think these are OK as-is, but if you're feeling charitable it'd be much appreciated if you switch to that.

See example at https://github.com/testing-cabal/testtools/blob/master/testtools/tests/matchers/test_basic.py#L115